internal/api/core/describe.go

```diff
@@ -13,6 +13,8 @@ import (
 	"github.com/edulinq/autograder/internal/util"
 )
 
+const TYPE_OVERRIDE_INDICATOR = "__TYPE_DESCRIPTION_OVERRIDE__"
```

Comment:
The name is pretty specific, but we would ideally want overrides to apply to different things (like fields as well).